x-pack/plugins/security_solution/public/common/containers/sourcerer/index.tsx

```diff
@@ -74,7 +86,8 @@ export const useInitSourcerer = (
       signalIndexName != null &&
```

Comment:
Could you please give me an example of how it happen in UI between Line86-90?

Reply:
Ok I renamed `signalIndexNameSelector` to `signalIndexNameSourcerer` so maybe that will help.

if the signal index is not loading, the `signalIndexName` is defined in useSignalIndex, but the `signalIndexNameSourcerer` has not been defined. This can happen the first time signal index is created. We also check to make sure that the Timeline sourcerer isn't already initiated and that default exists. Does that help?
```
    if (
      !loadingSignalIndex &&
      signalIndexName != null &&
      signalIndexNameSourcerer == null &&
      (activeTimeline == null || activeTimeline.savedObjectId == null) &&
      initialTimelineSourcerer.current &&
      defaultDataView.id.length > 0
    )
```